scripts/build.sh

```diff
@@ -85,7 +99,7 @@ function defaults() {
 	LTO="false"
 	BIN_PATH="unset"
 	RELEASE="fast"
-	SRC_PATH="$(dirname "$(dirname "$(realpath -s "$0")")")"
+	SRC_PATH="$(dirname "$(dirname "$(fullpath "$0")")")"
```

Comment:
I like the idea of assuming git is present, given it's part of our build instructions.
If someone downloads sources from SVN, they use `--src-path` to point the script at it, without having `git` present.  
If this script is merged upstream (into SVN), I suppose this would fail for people working out of the SVN repo instead of a git repo. But no sense in pre-solving that until the problem actually comes.

Reply:
Frankly, I forgot that someone is still using bare SVN for a moment :) It will work just fine for git-svn users though!

Reply:
Fixed in https://github.com/dreamer/dosbox-staging/pull/25/commits/decf81348a35c6d9674550ea71e5d7d340ef88de and https://github.com/dreamer/dosbox-staging/pull/25/commits/30443b95d2115ae48d090063a910131f36657eea

Reply:
Update: fails under MSYS2 with command-not-found, despite installing 'git' using pacman under MSYS2, because MSYS2 PATH is apparently not actually part of the system's PATH.  
This would work if there was a mingw-equivalent install for git, but there isn't.  So I've fallen back to a dead-simple "dirname, cd, and pwd" line that should work across all the distros.